angular.json

```diff
@@ -255,14 +269,19 @@
           },
           "configurations": {
             "production": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
```

Comment:
Is this necessary to include these budgets now, or is it an opt in. 
I'd opt for not introducing these budgets with this pull request and have a discussion on why we would need to add them in a separate issue.

Reply:
I agree lets remove them for now and come up with a good number for us in another PR

Reply:
This is done by the ng update schematic automatically because it is seen as go to. We should have this in place it is a warning nothing build breaking.

This is for the `demos app` and we should take this as measurement. because if we see  this warning we have done something wrong

Reply:
We have it already in place in all the other apps: `barista, components-e2e, dev`

Reply:
> We have it already in place in all the other apps: barista, components-e2e, dev

We did not have this before, these were added in your changeset. I'd still consider at least creating an issue to talk about the numbers in there, instead of blindly applying the defaults it ships with.